packages/playground/src/components/pull-request/sections/pull-request-comment-section.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { Text } from '@harnessio/canary'
+
+import { CheckCircleSolid } from '@harnessio/icons-noir'
+interface PullRequestMergeSectionProps {
+  commentsInfo: { header: string; content?: string | undefined; status: string }
+}
+const PullRequestCommentSection = ({ commentsInfo }: PullRequestMergeSectionProps) => {
+  return (
+    <div className="py-4  border-b">
```

Comment:
why extra spaces between class names? Is it due to formatting?

Reply:
fixed